res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -68,42 +74,16 @@ limitations under the License.
         }
 
         .mx_MessageActionBar_maskButton {
-            --size: 24px;
-            width: var(--size);
-            height: var(--size);
+            width: var(--ThreadPanel_header-button-size);
+            height: var(--ThreadPanel_header-button-size);
 
             &::after {
-                mask-size: var(--size);
+                mask-size: var(--ThreadPanel_header-button-size);
                 mask-image: url("$(res)/img/element-icons/message/overflow-large.svg");
             }
         }
     }
 
-    .mx_ThreadPanel_button {
```

Comment:
Added with https://github.com/matrix-org/matrix-react-sdk/pull/7063/files#diff-9ee2dc5439d26e16a07dab8a3da737096f606b879a3da2c3c3411d7fa560628aR274. It has been superseded with [ThreadListContextMenu.tsx#L85](https://github.com/matrix-org/matrix-react-sdk/blob/efc36acf933406a7f29a1aef7fcd6faa20e182dd/src/components/views/context_menus/ThreadListContextMenu.tsx#L85).